script/run_tests_in_docker

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
```

Comment:
> I think we cannot assume it's available on the developer machine.

It should be. The convenience is huge - it simplifies building, running, the configuration is readable, tweakable (rather than hard-coded parameters in a Rakefile or on the command line), it automatically detects if a build is needed, you can run multiple services at once (JRuby), it properly names containers, variable substitution ... lots of goodies to simplify development.

It takes away the brain-work. It's hard NOT to use it. It would be like not using Rake for Ruby projects (and just using scripts instead).

You install it once, and it's on your desktop forever. Anyone potentially ever using docker will highly likely also install Docker Compose. So why not get it over with.

If someone really does not want Docker Compose ... they probably don't care about Docker either, so they can just use non-docker tests and Travis. (Running tests through docker is optional to begin with).